### PR TITLE
(8210): Updated test resources to add import support

### PR DESCRIPTION
**Pull request by zhiqiangjin**
Test passed:

1. account_user
2. version
3. gateway
4. transit_vpc
5. tgw

### CODE REVIEW

File: aviatrix/resource_account_test.go [hunk 1]
```diff
@@ -77,25 +79,30 @@ func TestAccAviatrixAccount_basic(t *testing.T) {
...
-							"aviatrix_account.aws", "aws_secret_key",
-							os.Getenv("AWS_SECRET_KEY")),
+							resourceName, "aws_secret_key", os.Getenv("AWS_SECRET_KEY")),
 					),
 				},
+				{
+					ResourceName:            resourceName,
+					ImportState:             true,
+					ImportStateVerify:       true,
+					ImportStateVerifyIgnore: importStateVerifyIgnore,
```
Comment (arvindsree) [inline aviatrix/resource_account_test.go]: This is list right, I think you should just say ["access_key"]

Comment (zhiqiangjin) [reply aviatrix/resource_account_test.go]: Tried, did not work.

File: aviatrix/resource_spoke_vpc_test.go [hunk 2]
```diff
@@ -74,11 +74,13 @@ func TestAccAviatrixSpokeGw_basic(t *testing.T) {
 						resource.TestCheckResourceAttr(resourceName, "subnet", os.Getenv("AWS_VPC_NET")),
 						resource.TestCheckResourceAttr(resourceName, "vpc_reg", os.Getenv("AWS_REGION")),
 						resource.TestCheckResourceAttr(resourceName, "enable_nat", "no"),
-						resource.TestCheckResourceAttr(resourceName, "tag_list.#", "2"),
```
Comment (arvindsree) [inline aviatrix/resource_spoke_vpc_test.go]: Lets add these. You can ignore in import state

Comment (arvindsree) [reply aviatrix/resource_spoke_vpc_test.go]: or you can add single tag

**Pull request merged**